### ISSUE TITLE
Scripts to generate corrupted data

### ISSUE THREAD

**Issue by fushuhao6** [yueatsprograms/ttt_cifar_release#1]
First of all, thank you so much for your work! The paper is really impressive!
Could you also provide the scripts to produce corrupted CIFAR data for testing?
Besides, is it possible to use other datasets to test its adaptability (train on CIFAR-10 and test on other datasets)? Say, 32 x 32 ImageNet dataset?

**Comment by yueatsprograms** [comment]
Thank you for your kind comments. Sorry for my tardiness in getting the instructions up, but now there's a readme which includes the link to the CIFAR data. For your second question, yes it is possible to use other 32 x 32 image datasets once you reorganize the classes to match the labels of your training data, but I haven't tried those.
